pages/main/faq.md

```diff
@@ -0,0 +1,55 @@
+slug: faq
+layout: documentation
+---
+
+# What is AsteroidOS?
+AsteroidOS is an open source Linux distribution that runs on a watch. It uses [Qt](http://www.qt.io/) Qt and QML to provide the graphical interface.
+
+# Is it Android?
+No. AsteroidOS uses [libhybris](https://en.wikipedia.org/wiki/Hybris_(software)) to simplify porting to most Android and Wear OS watches, but it is not Android.
+
+# Can I run WearOS applications?
+No.  [WearOS](https://en.wikipedia.org/wiki/Wear_OS) is a version of Android that runs on wearable devices, but AsteroidOS is a Linux distribution that does not run Android and therefore cannot run either Android or WearOS applications.
+
+# What are the differences between AsteroidOS and WearOS?
+One significant difference is that AsteroidOS is open source software, while WearOS is not.  This means that if you would like to change something, and you are a developer, you can [build the software]({{rel 'wiki/building-asteroidos'}}) yourself.  Or perhaps you would like to [create your own watchface]({{rel 'wiki/watchfaces-creation'}}).  Another significant difference is that unlike WearOS, AsteroidOS tries to make the watch useful even without pairing it with a phone, while WearOS watches won't even start running until they are paired with a phone.
+
+# Does AsteroidOS have any tracking features?
+No. AsteroidOS does not require connecting any accounts to use it, and does not even collect any telemetry. Nothing on the default installation is able to connect to the internet.  While it is possible to [connect an AsteroidOS watch to the internet]({{rel 'wiki/ip-connection'}}) this must be explicitly done by the user.
+
+# Can I run AsteroidOS on my watch?
+Maybe.  Check the [Install]({{rel 'install'}}) page to see if your watch is listed.
+
+# Which features do/don't work on my watch?
+Check the [features table]({{rel 'install/features'}}) to see all of the watches and which features they support.
+
+# My watch isn't listed.  Can I still run AsteroidOS?
+Maybe in the future. See the [porting status]({{rel 'wiki/porting-status'}}) table for details on what kinds of watches might be supported in the future and what the general requirements are for running AsteroidOS.
+
+# Will I be able to revert to the previous operating system after installing AsteroidOS on my watch?
+Yes, if you choose the "temporary install" option.  See the [installation FAQ]({{rel 'install#FAQ'}}) for details.
+
+# Where can I find a phone synchronization app?
+A phone is not required to use AsteroidOS, but a synchronization client for Android phones named "AsteroidOSSync" is available for [download on F-Droid](https://f-droid.org/packages/org.asteroidos.sync/). Ubuntu Touch users can download ["Telescope" from OpenStore](https://open-store.io/app/telescope.asteroidos).  There is currently no app for iPhone, however notifications can be shared from an iPhone to the watch.  See [this page]({{rel 'wiki/synchronization-clients'}}) for details.
```

Comment:
This sounds a bit like a defense instead of a feature here. To prepare for missing clients for some systems? I think it was made clear above that AOS does not require a phone to work. Tho it is our core philosophie and worth to repeat. Imo this section should directly start into "A synchronization client for..." to stay on topic.

Reply:
That sounds nice.